src/main/java/rx/internal/operators/OnSubscribeAmb.java

```diff
@@ -343,10 +344,13 @@ public void unsubscribeOthers(AmbSubscriber<T> notThis) {
         }
 
     }
-
-    private final Iterable<? extends Observable<? extends T>> sources;
-    private final Selection<T> selection = new Selection<T>();
-
+    
+    //give default access instead of private as a micro-optimization 
+    //for access from anonymous classes below
+    final Iterable<? extends Observable<? extends T>> sources;
```

Comment:
And generates an accessor method in that outer class, the count of which (methods) is a big deal on Android. So :+1:.